constants.py

```diff
@@ -6,6 +6,8 @@
 
 TRACER_ATTRIBUTE = "pytype_tracer"
 
+SAMPLE_CODE_FOLDER_NAME = "examples"
```

Comment:
The constants is here to set where the folder path is where the sample codebase + tests are. It will be used so that the decorator appender searches for test files in that folder (+ its subfolders). It probably won't be the examples folder in the future, but currently it is the folder with contains sample code + sample tests.

Reply:
The examples folder is intended for literal examples, i.e. how our API works on test code.
It is not meant not to be used for testing purposes (if I understand your remark correctly?)

Reply:
The constant is used in the main()-function. In the future, the constant contains the folder name in which the sample code & tests reside #9 . It is not used in any unit test.

Reply:
Ah I see now. This is a good idea, and I suppose we can keep it for now, but quite soon, we will need to add a command line interface for each of the stages in our workflow (fetching + hooking, tracing, stubgen). 

The `main.py` file will be used as the entrypoint, meaning the `TracerDecoratorAppender` will accept its arguments from the command line. This means the code in `main.py`, as it stands, will be entirely replaced.

I have this on my feature branch currently, simply for the fetch code.

```
λ python main.py --help    
Usage: main.py [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  fetch  download repositories and apply tracing decorators  
```

```
λ python main.py fetch --help
Usage: main.py fetch [OPTIONS]

  download repositories and apply tracing decorators

Options:
  -u, --url TEXT              URL to a repository  [required]
  -f, --format [Git|Archive]  Indicate repository format explicitly
  -o, --output PATH           Download output path  [required]
  --help                      Show this message and exit. 
```